### PR TITLE
Ui branch

### PR DESCRIPTION
**Pull request by HMCodingSnowMan**

### CODE REVIEW

File: app/searchWindows.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+class searchWindows(object):
+    def searchBoxGo(self):
+        msg = "Enter a search Query, Then push OK"
+        title = "Item Finder"
+        fieldValues = []  # we start with blanks for the values
+        fieldValues = easygui.enterbox(msg,title, fieldValues)
+
+        # make sure that none of the fields was left blank
+        while 1:
```
Comment (Tysterman74) [inline app/searchWindows.py]: Use True instead of 1

File: app/searchWindows.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        
+    def msgBoxGo(self,fieldValue):
+        msgWindow="Searching for %s" %str(fieldValue)
+        easygui.msgbox(msgWindow)
+        #message window saying we are searching for the search term
+
+    def resultsBoxGo(self):
+        easygui.msgbox("Here are the results")
+
+    def ynBoxGo(self):
```
Comment (Tysterman74) [inline app/searchWindows.py]: Rename this function to exportToExcelGo

File: app/searchWindows.py (hunk 1 above)
Comment (HMCodingSnowMan) [reply app/searchWindows.py]: pushed

File: app/searchWindows.py (hunk 2 above)
Comment (HMCodingSnowMan) [reply app/searchWindows.py]: change made

**Pull request merged**